src/main/java/it/free/Serv1.java

```diff
@@ -0,0 +1,16 @@
+package it.free;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class Serv1 extends HttpServlet {
```

Comment:
not obvious servlet name. If you want to name like Servlet, use full word "Servlet"